vite.config.ts

```diff
@@ -1,6 +1,7 @@
 import reactRefresh from '@vitejs/plugin-react-refresh'
 import tsconfigPaths from 'vite-tsconfig-paths'
 import { defineConfig } from 'vite'
+import path from 'path'
```

Comment:
What exactly did u try to reach?

Reply:
I was trying to add fonts in vite config

Reply:
fixed